code/modules/admin/banjob.dm

```diff
@@ -140,3 +140,58 @@ DEBUG
 				jobban_savebanfile()
 			return 1
 	return 0
+
+/mob/verb/displayjobbans()

```

Comment:
Perhaps include a clickable link to this on the job preferences screen, if the user has active job bans?